.drone.star

```diff
@@ -734,8 +776,15 @@ def buildPhoenix():
 		'pull': 'always',
 		'commands': [
 			'yarn dist',
-			'cp tests/drone/config.json dist/config.json'
-		]
+			'cp tests/drone/config.json dist/config.json',
+			'mkdir -p /srv/config',
+			'cp -r /var/www/owncloud/phoenix/tests/drone /srv/config',
+			'ls -la /srv/config/drone'
```

Comment:
extra debug line?

Reply:
Not exactly, It would be nice to see if the files are copied in the logs.